realm/realm-library/src/main/java/io/realm/RealmQuery.java

```diff
@@ -1502,15 +1510,19 @@ public long count() {
      */
     @SuppressWarnings("unchecked")
     public RealmResults<E> findAll() {
-        Collection collection = new Collection(realm.sharedRealm, query);
-        return createRealmResults(collection);
+        return createRealmResults(query, null, null, true);
     }
 
     /**
-     * @deprecated use {@link #findAll()} instead.
+     * Finds all objects that fulfill the query conditions and sorted by specific field name.
```

Comment:
Checked in the dic, they are the same i think? Anyway, the javadoc is not new :) I will change it in another PR if needed.